Say Their Names/Source/View/Home/Person/PersonPhotoCell.swift

```diff
@@ -8,19 +8,63 @@
 
 import UIKit
 
-class PersonPhotoCell: UICollectionViewCell {
-
-    override func awakeFromNib() {
-        super.awakeFromNib()
-        // Initialization code
+final class PersonPhotoCell: UICollectionViewCell {
+    
+    var personImage: UIImage? {
+        didSet {
```

Comment:
Latest s updates to Swift 5 suggest that we shouldn’t be using `didSet` unless we are specifically working with logic that involves `oldValue`.

Also, we will be using the `ImageService` to handle threading and setting the image

Reply:
I wasn't aware of this, thanks. Then, should personImageView be exposed to be internal?

Reply:
I got it, you meant I shouldn't be sending it to the main queue, since it is responsibility of `ImageService`. I've update it. Although I find safer to send it to the main queue in the actual implementation of the change of the image, I think it is less error-prone than expecting the call to the function to be made in the correct queue